buildSrc/src/main/groovy/com/carrotsearch/gradle/junit4/RandomizedTestingTask.groovy

```diff
@@ -211,7 +223,7 @@ class RandomizedTestingTask extends DefaultTask {
 
         DefaultLogger listener = null
         ByteArrayOutputStream antLoggingBuffer = null
-        if (logger.isInfoEnabled() == false) {
+        if (false == logger.isInfoEnabled()) {
```

Comment:
I still see lots of arbitrary syntactical changes like this. Every extra line of output makes something that must be looked at. Without these minor changes, I think this PR would only be a dozen lines or so and would be very quick to review.

Reply:
I don't have a problem omitting them if that's the consensus, my take on it is that the cost of making and reviewing these is worthwhile if they improve the readability of the code, as it eventually pays off.